### PR TITLE
hvp keymap for Cornelius, updated ut47.2 and userspace

### PR DESCRIPTION
**Pull request by hvpkod**
## Description

New keymap for Cornelius keyboard and updated ut47.2 keymap. Changes to hvp userspace.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: users/hvp/config.h [hunk 1]
```diff
@@ -0,0 +1 @@
+#define LONG_TAPPING_TERM 1000
```
Comment (drashna) [inline users/hvp/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15


```suggestion
#pragma once

#define LONG_TAPPING_TERM 1000
```

File: users/hvp/per_key.c [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#include per_key.h
```
Comment (drashna) [inline users/hvp/per_key.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: users/hvp/per_key.c (hunk 2 above)
Comment (hvpkod) [reply users/hvp/per_key.c:1]: Just added one now based on the template.

File: users/hvp/config.h (hunk 1 above)
Comment (drashna) [reply users/hvp/config.h:17]: This still needs an include header.

**Review by drashna — APPROVED** [review]

File: users/hvp/config.h (hunk 1 above)
Comment (hvpkod) [reply users/hvp/config.h:18]: Fixed

File: users/hvp/per_key.c [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+  * GNU General Public License for more details. 
+  * 
+  * You should have received a copy of the GNU General Public License 
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+  */ 
+
+#pragma once
+#include per_key.h
```
Comment (fauxpark) [inline users/hvp/per_key.c:18]:
```suggestion
```
This is an empty file, please delete it.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**